Classes/Domain/Model/Glossary.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sitegeist\LostInTranslation\Domain\Model;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Neos\Flow\Annotations as Flow;
+use League\Csv\Writer;
+
+/**
+ * @Flow\Entity
+ * @ORM\Table(uniqueConstraints={@ORM\UniqueConstraint(name="languageCombination", columns={"sourceLanguageKey", "targetLanguageKey"})})
+ */
+class Glossary
+{
+    /**
+     * @var string
+     */
+    public string $sourceLanguageKey;
+
+    /**
+     * @var string
+     */
+    public string $targetLanguageKey;
+
+    /**
+     * @var \DateTimeImmutable|null
+     * @ORM\Column(nullable=true)
+     */
+    public $synchronizationDate;
+
+    /**
+     * @var string|null
+     * @ORM\Column(nullable=true)
+     */
+    public $synchronizationIdentifier;
+
+    /**
+     * @var \DateTimeImmutable
+     */
+    public $modificationDate;
+
+    /**
+     * @phpstan-var Collection<int, GlossaryEntry>
+     * @var Collection<GlossaryEntry>
+     * @ORM\OneToMany(targetEntity="Sitegeist\LostInTranslation\Domain\Model\GlossaryEntry", mappedBy="glossary", cascade={"persist"})
```

Comment:
I think we are missing cascade update here. This is why, after updating a glossary entry, it would still show the old content.